gitlab-examples/gitlab-pipline/README.md

```diff
@@ -0,0 +1,23 @@
+# GitLab pipeline example
+This is an example of a GitLab pipeline which uses JFrog CLI to integrate with artifactory.
+This sample uses a gradle project, builds it, and push the artifact to artifactory, including build info, environment variables and SBOM.
```

Comment:
```suggestion
This sample uses a Gradle project, builds it, and pushes the artifact to Artifactory, including build info, environment variables, and SBOM.
```

Reply:
Does it publish an SBOM? How come?

Reply:
I assume it does, as I am also publishing BuildInfo, and scanning with XRay, but maybe I do not understand the difference yet. 
Please suggest rephrasing, if you think this is wrong.